common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/ElasticSearchExecutions.scala

```diff
@@ -40,7 +42,12 @@ trait ElasticSearchExecutions {
     result.failed.foreach { e =>
       val elapsed = System.currentTimeMillis() - start
       val markers = MarkerContext(durationMarker(elapsed))
-      Logger.error(s"$message - query failed after $elapsed ms: ${e.getMessage} cs: ${e.getCause}")(markers)
+      e match {
+        case ElasticNotFoundException => Logger.error(s"$message - query failed: Document not Found")(markers)
+        case ElasticException(error) => Logger.error(s"$message - query failed after: ${error.reason} type: ${error.`type`}")(markers)
```

Comment:
```suggestion
        case ElasticException(error) => Logger.error(s"$message - query failed because: ${error.reason} type: ${error.`type`}")(markers)
```